### PR TITLE
Resources for VN-412

### PR DESCRIPTION
**Pull request by votuanphong1983**
Add some resources for [VN-412 ticket](https://freedom.myjetbrains.com/youtrack/issue/VN-280)

### CODE REVIEW

File: zh/index.json [hunk 1]
```diff
@@ -16,7 +16,8 @@
     "update": "更新"
   },
   "error": {
-    "not_found": "哎呀!這裡甚麼都沒有"
+    "not_found": "哎呀!這裡甚麼都沒有",
+    "connection_fail": "Connection error"
```
Comment (pandafox) [inline zh/index.json]: "connection_fail": "連線問題"

File: cs/index.json [hunk 2]
```diff
@@ -123,5 +124,9 @@
     "verify": "VERIFY",
     "payment_test_failure": "Failed to enqueue test payment!",
     "payment_not_found": "User payment not found! Please update payment method at Freedom! dashboard."
+  },
+  "instruction": {
+    "instruction": "instruction",
```
Comment (pandafox) [inline cs/index.json]: Instruction

File: zh/index.json [hunk 3]
```diff
@@ -123,5 +124,9 @@
     "verify": "驗證",
     "payment_test_failure": "無法執行驗證付款!",
     "payment_not_found": "付款方式還未設定! 請先到Freedom!增加一個付款方式"
+  },
+  "instruction": {
+    "instruction": "instruction",
+    "instruction_tooltip": "Some tips to improve mining performance or fix mining issues"
```
Comment (pandafox) [inline zh/index.json]: It's a bit odd that we call it Instruction, more like Instruction Manual?

```
    "instruction": "使用說明",
    "instruction_tooltip": "提高采礦性能或修復採礦問題的一些技巧"
```

**Review by pandafox — DISMISSED** [review]

**Review by huy2nhan — APPROVED** [review]
Looks good

**Pull request merged**